src/app/components/Grid/index.jsx

```diff
@@ -7,7 +7,7 @@ import {
 } from '@bbc/gel-foundations/breakpoints';
 import { C_GHOST } from '@bbc/psammead-styles/colours';
 
-const group4WrapperMaxWidth = `63rem`; // 1008px
+export const group4WrapperMaxWidth = `63rem`; // 1008px
```

Comment:
I'm not sure how I feel about exporting a value from a component for use elsewhere. It might over-complicate things a little?

Is there any reason these values are hardcoded and not taken from `@bbc/gel-foundations/breakpoints`?
```js
GEL_GROUP_4_SCREEN_WIDTH_MIN; // 63rem
GEL_GROUP_5_SCREEN_WIDTH_MIN // 80rem;
```
I guess these are min values rather than max, but perhaps they could be used with a comment to explain why this is the case? I may have missed previous conversations about this though.

Reply:
Agree, we can use directly the `gel-foundations` values, rather than declaring new constants with the same values. For some reason, we didn't think in the past that those were already in the package.